effekt/shared/src/main/scala/effekt/generator/js/TransformerMonadic.scala

```diff
@@ -110,11 +113,13 @@ trait TransformerMonadic extends Transformer {
   def toJS(p: Param): JSName = nameDef(p.id)
 
   def toJS(e: core.Extern)(using DeclarationContext, Context): js.Stmt = e match {
-    case Extern.Def(id, tps, cps, vps, bps, ret, capt, body) =>
-      js.Function(nameDef(id), (vps ++ bps) map toJS, List(js.Return(toJS(body))))
+    case Extern.Def(id, tps, cps, vps, bps, ret, capt, ExternBody(featureFlag, contents)) =>
+      js.Function(nameDef(id), (vps ++ bps) map toJS, List(js.Return(toJS(contents))))
 
-    case Extern.Include(contents) =>
+    case Extern.Include(ff, contents) if ff.matches(jsFeatureFlags) =>
       js.RawStmt(contents)
+
+    case Extern.Include(_, _) => js.RawStmt("") // ignore, not meant for us
```

Comment:
Can this still happen? I thought selection happens earlier and only leaves those that we care about.

Reply:
Or just drop the check altogether. I'll do the latter.